pkg/controllers/interface_filter.go

```diff
@@ -30,3 +30,13 @@ func getDefaultNetwork(networks []v1.Network) *v1.Network {
 func isDefaultNetwork(net v1.Network) bool {
 	return net.Pod != nil || (net.Multus != nil && net.Multus.Default)
 }
+
+func FilterInterfacesWithNoName(ifaces []v1.VirtualMachineInstanceNetworkInterface) []v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
btw since it is used in the same package, maybe it can be lower cased first character ?

Reply:
but it is used internally only, we don't expose stuff that is not need to be exposed

Reply:
The public method are the methods that the interace_file exposes. I want it to be clear.
It is even [considered a good practice](https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742) to have the unit tests in a separate package, so it will be clear what is the API of the tested file.

Reply:
But we don't have it on a separate package atm.
also API is for stuff that is used outside of the module, not inside.
else i would suggest for example that once we move unit tests to their own package
then we can expose it

Reply:
moved the filter tests to a separate package.